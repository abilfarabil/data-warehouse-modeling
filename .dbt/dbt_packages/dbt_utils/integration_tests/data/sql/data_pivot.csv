version https://git-lfs.github.com/spec/v1
oid sha256:bb404216ef9e331790a43822298f225bf75b1904865de7e962465ce45a460554
size 42
