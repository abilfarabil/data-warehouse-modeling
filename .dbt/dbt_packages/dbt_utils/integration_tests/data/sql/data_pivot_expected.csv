version https://git-lfs.github.com/spec/v1
oid sha256:186e6bc556fb33f7c12c331c391b076c25266c344d610153665e8b5d1d1e23f7
size 27
